src/modules/hardwareWallet/components/HWReconnect/HWReconnect.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Dialog from 'src/theme/dialog/dialog';
+import Box from '@theme/box';
+import BoxHeader from '@theme/box/header';
+import BoxContent from '@theme/box/content';
+import Illustration from '@common/components/illustration';
+import WalletVisual from '@wallet/components/walletVisual';
+import { useCurrentAccount } from '@account/hooks';
+import styles from './HWReconnect.css';
+
+const HWReconnect = () => {
+  const [currentAccount] = useCurrentAccount();
+  const { name, address } = currentAccount.metadata;
+
+  return (
+    <Dialog className={styles.container} hasClose hasBack size="sm">
+      <Box className={styles.wrapper}>
+        <BoxHeader className={styles.header}>
+          <div>
+            <Illustration name="hwReconnection" />
+          </div>
+          <h4>Reconnect to hardware wallet</h4>
+        </BoxHeader>
+        <BoxContent className={styles.content}>
+          <div className={styles.details}>
+            <p>Your hardware wallet is disconnected.</p>
+            <p>Please reconnect your hardware wallet to sign this transaction</p>
+          </div>
+          <div className={styles.account}>
+            <WalletVisual address={address} size={32} />
+            <div className={styles.name}>
```

Comment:
Unnecessary tag?
Could as well be
`<span className={styles.name}>{name}</span>`

and have display flex or box on the class rule. Same for address

Reply:
@oskarleonard That's possible, however I prefer to use some elements as they were semantically designed. Since spans are inline elements, I prefer wrapping them in div when I need them to occupy the block width